### ISSUE TITLE
NoSuchMethodError: The method 'querySelector' was called on null.

### ISSUE THREAD

**Issue by SwissCheese5** [Hexer10/youtube_explode_dart#27]
```
NoSuchMethodError: The method 'querySelector' was called on null.
Receiver: null
Tried calling: querySelector("a")
```

Calling: 
```dart
(await YoutubeExplode().getChannelIdFromVideo(video.id))
```

**Comment by SwissCheese5** [comment]
For more context, here's the entire code:

```dart
  static Future<List<RequestTrack>> search(String query) async {
    var results = <RequestTrack>[];

    var videos = await YoutubeExplode().searchVideos(query);
    for (var i = 0; i < videos.length; i++) {
      if (i >= 10) break;

      var video = videos[i];

      results.add(RequestTrack(
        originID: video.id,
        originURL: 'https://www.youtube.com/watch?v=${video.id}',
        publishedAt: video.uploadDate.millisecondsSinceEpoch,
        title: video.title,
        description: video.description,
        channelTitle: video.author,
        channelURL: 'https://www.youtube.com/user/${(await YoutubeExplode().getChannelIdFromVideo(video.id))}',
        duration: video.duration.inMilliseconds,
        views: video.statistics.viewCount,
        tags: video.keyWords,
        thumbnailURL: video.thumbnailSet.standardResUrl,
        fullCoverURL: video.thumbnailSet.maxResUrl,
        blurHash: await Utils.genBlurHash(video.thumbnailSet.lowResUrl),
      ));
    }

    return results;
  }
```

**Comment by Hexer10** [comment]
This should get fixed in `1.0.0`  (which includes several breaking changes (mostly syntax) ), I've already pushed a beta version (1.0.0-beta) but it doesn't include the search client yet, I plan on shipping it in `1.0.1-beta`
